remote_vm_setup_vars.yaml

```diff
@@ -0,0 +1,48 @@
+# Here we put the ansible vars to override some defaults in the vm-setup
+# playbooks/roles, ref https://github.com/metal3-io/metal3-dev-env
+
+# Currently this is required because of hard-coded node-name expectations in the
+# openshift-installer terraform templates
+ironic_prefix: "{{ ironic_prefix }}"
+
+# We enable more memory and masters in dev-scripts compared to the minimal setup
+# in metal3-dev-env
+num_masters: 1
+num_workers: 0
+flavors:
+  master:
+    memory: "{{ lookup('env', 'MASTER_MEMORY') | default('16384', true) }}"
+    disk: "{{ lookup('env', 'MASTER_DISK') | default('30', true) }}"
+    vcpu: "{{ lookup('env', 'MASTER_VCPU') | default('8', true) }}"
+    extradisks: false
+
+  worker:
+    memory: "{{ lookup('env', 'WORKER_MEMORY') | default('8192', true) }}"
+    disk: "{{ lookup('env', 'WORKER_DISK') | default('20', true) }}"
+    vcpu: "{{ lookup('env', 'WORKER_VCPU') | default('4', true) }}"
+    extradisks: false
+
+remote_cluster_network_cidr_v4: "{{ lookup('env', 'REMOTE_CLUSTER_SUBNET_V4') }}"
```

Comment:
I wonder if we could avoid this duplication by making the variable keys configurable in the existing vars file?

All we need is a way to rename one of the flavors, and have that influence the node/network name, which could simplify this quite a bit?